ui/src/views/DashboardView/DashboardView.tsx

```diff
@@ -37,7 +43,7 @@ const DashboardView = () => {
   const { loading: loadingZaaktypen, error: errorZaaktypen } = useAsync(async () => {
     if (!selectedCatalogus) return;
 
-    const zaaktypenEndpoint = `catalogi/zaaktypen/?status=alles${
+    const zaaktypenEndpoint = `catalogi/zaaktypen/?status=alles&page=1${
```

Comment:
Is this in preparation for this endpoint to support pagination? Should it maybe be added later?

Reply:
I think this is a mistake, thanks for noticing!